ui/static/ui/css/mit-lore.css

```diff
@@ -546,6 +561,10 @@ ul.repos {
     transform: translateY(-27%);
 }
 
+.nav-tabs {
+    border-bottom:1px solid #ddd !Important;
```

Comment:
These styles were not getting applied properly -- I think there are default styles somewhere else, or perhaps a loading order issue that broke the intended cascade. See lore-ci to see what I mean. 
<img width="164" alt="screenshot 2015-09-18 14 52 14" src="https://cloud.githubusercontent.com/assets/882377/9968160/e35dca22-5e14-11e5-8303-cbe5f7bc960e.png">